config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -300,46 +252,40 @@ presubmits:
       testgrid-num-failures-to-alert: "10"
       testgrid-dashboards: google-gce
     labels:
-      preset-service-account: "true"
       preset-k8s-ssh: "true"
-      preset-pull-kubernetes-e2e: "true"
-      preset-pull-kubernetes-e2e-gce: "true"
+      preset-dind-enabled: "true"
     decorate: true
     decoration_config:
-      timeout: 105m
+      timeout: 110m
     path_alias: k8s.io/kubernetes
     extra_refs:
     - org: kubernetes
-      repo: release
+      repo: kops
       base_ref: master
-      path_alias: k8s.io/release
```

Comment:
why did this mount release? is ok to swap release by kops?

Reply:
Yes, kops implementes krel to build and stage kube builds

https://github.com/kubernetes/kops/blob/master/tests/e2e/kubetest2-kops/builder/build.go#L67